packages/editor-ui/src/views/NodeView.vue

```diff
@@ -3367,6 +3367,7 @@ export default mixins(
 			dataPinningEventBus.$off('pin-data', this.addPinDataConnections);
 			dataPinningEventBus.$off('unpin-data', this.removePinDataConnections);
 			nodeViewEventBus.$off('saveWorkflow', this.saveCurrentWorkflowExternal);
+			this.workflowsStore.setWorkflowId(PLACEHOLDER_EMPTY_WORKFLOW_ID);
```

Comment:
Why was this needed? Here and at L3379.

Reply:
Otherwise last active workflow ID stuck in the store. In my thinking, if the user is not on the workflow page there should not be active workflow ID, right? 

This stuck ID caused some trouble on the workflow list page.

Reply:
Thank you, makes sense.